### PR TITLE
feat(suite-native): token discovery analytics

### LINKED ISSUE #11487: Analytics - Report ETH tokens of discovered accounts
## Description
At the moment, when there is a new account created from connected device discovery, we report only the network symbol to the analytics server. There should be also valid (by token definitions) token symbols a token contracts related to that account, similar as we do for the portfolio tracker imported account. 

Current shape of the analytics message dispatched:
```ts
{
    type: EventType.CoinDiscoveryNewAccount;
    payload: {
        symbol: NetworkSymbol;
        path: string;
        type: AccountType;
    };
}
```

## Acceptance criteria
Token symbols and contracts are reported to analytics for every device discovered account.

### PR DESCRIPTION
**Pull request by PeKne**
## Description
- array of token symbols and contracts included in discovered accounts reported to analytics
## Related Issue

Resolve #11487 

Analytics Changelog draft: [Notion link](https://www.notion.so/satoshilabs/Changelog-Mobile-app-946307159a294bcdb90b94e90eefd292?pvs=4#b871f5e5e0c346478b3a4fd88b840127)

### CODE REVIEW

File: suite-native/discovery/src/discoveryThunks.ts [hunk 1]
```diff
@@ -129,8 +132,8 @@ const getCardanoSupportedAccountTypesThunk = createThunk(
     },
 );
 
-const addAccountsByDescriptorThunk = createThunk(
```
Comment (vytick) [inline suite-native/discovery/src/discoveryThunks.ts:132]: nice catch 👍

**Review by vytick — APPROVED** [review]
LGTM

File: packages/analytics/src/types.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export type Event = {
     type: string;
     timestamp?: string;
     payload?: {
-        [key: string]: string | string[] | number | number[] | boolean | null;
+        [key: string]: string | string[] | number | number[] | boolean | object | null;
```
Comment (matejkriz) [inline packages/analytics/src/types.ts:23]: Are we sure that `object` is ok as payload? Would be nice to double check with data team (cc @kolomaznikt) that they can work with it.

Comment (PeKne) [reply packages/analytics/src/types.ts:23]: I already texted him on Slack. I will let you know what is the outcome of the discussion ;).

Comment (PeKne) [reply packages/analytics/src/types.ts:23]: @matejkriz you were right. The Keboola is unable to parse `object` attributes. So we will have to change to solution a little. Until there is a new format proposal from product+data team, I'm reopening this issue and moving it back to TODO.

**Pull request merged**